resources/js/utils/database.js

```diff
@@ -1,6 +1,10 @@
 const Datastore = require('nedb');
 const {app} = require('electron').remote;
 
+function modulo(_id, bounds) {
```

Comment:
_id parameter confuse, because is the same as _id database unique identifier

Reply:
I pushed a new commit with the requested changes.